### PR TITLE
#26: use dotenv to configure poster/metadata service location

### PR DESCRIPTION
**Pull request by alexscheitlin**
### Description

Use dotenv to configure poster and metadata service.

### Related Issues

#26 Environment Setup

### Checklist

- [x] Have you added tests where necessary? Do all the test pass? 
- [x] Have you added descriptive comments to your code?
- [x] Have you updated the documentation related to this propo

### CODE REVIEW

**Comment by alexscheitlin** [comment]
Where do we set the variables for the deployment? As GitHub secrets?

**Comment by nikzaugg** [comment]
> Where do we set the variables for the deployment? As GitHub secrets?

Yes, I would say so, right @rschlaefli ?

**Comment by rschlaefli** [comment]
I think we would set them in Heroku, as Docker images are normally configured where they run, not where they are built. I can look it up :)

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "@nestjs/graphql": "7.0.1",
     "@nestjs/platform-express": "7.0.1",
     "apollo-server-express": "2.11.0",
+    "dotenv": "^8.2.0",
```
Comment (rschlaefli) [inline package.json]: Pin :)

Comment (alexscheitlin) [reply package.json]: Oh, I forgot that :) Thanks for the reminder (Y)

**Comment by alexscheitlin** [comment]
> I think we would set them in Heroku, as Docker images are normally configured where they run, not where they are built. I can look it up :)

Thanks. They are now set to:
```
METADATA_SERVICE_URL=https://kwiz-metadata-service.herokuapp.com/
POSTER_SERVICE_URL=https://kwiz-poster-service.herokuapp.com/
```

**Comment by alexscheitlin** [comment]
> The lockfile will need to be regenerated (so the pinning is reflected). But LGTM otherwise!

As well as running `npm install` as removing the `package-lock.json` and running `npm install` does not change anything with the axios dependency.

**Pull request merged**